src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -564,10 +566,10 @@
         label = "The Respondent's home address",
         hint = "If the respondent is to be served at their home address, enter the home address here and as the service "
             + "address below",
-        typeOverride = TextArea,
+        typeOverride = Address,
         access = {DefaultAccess.class}
     )
-    private String respondentHomeAddress;
+    private AddressGlobalUK respondentHomeAddress;
```

Comment:
as you are changing type you will need to bump case type from nofaultdivorce 6 to 7 otherwise elasticsearch will break